### PR TITLE
Login Context

### PR DESCRIPTION
**Pull request by rushikesh3299**
Context for login with minor code refactors

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
 import { Login, Signup, ProductPage, Home, ErrorPage } from "./pages/index";
 import Mockman from "mockman-js";
 
 export const App = () => {
+  const location = useLocation();
+
   return (
     <div className="App">
-      <Navbar />
+      {location.pathname != "/login" && location.pathname != "/signup" && (
```
Comment (t007rushi) [inline src/App.js:12]: can create separate array for unwanted paths

File: src/pages/authpages/Login.js [hunk 2]
```diff
@@ -1,17 +1,49 @@
...
       <h3>Enter Your info below to Login</h3>
-      <div className="login-inputs">
-        <input type="text" className="userid" placeholder="User ID" />
-        <input type="password" name="passwd" id="" placeholder="Password" />
-        <button className="login-btn">Login</button>
+      <form
+        className="login-inputs"
+        onSubmit={(e) => {
+          e.preventDefault();
+          loginHandler(loginFormData, userData, setUserData, navigate);
```
Comment (t007rushi) [inline src/pages/authpages/Login.js:24]: don't pass setter fun into function call

**Review by t007rushi — APPROVED** [review]
Look for suggestions else can go ahead

File: src/pages/authpages/Login.js [hunk 3]
```diff
@@ -1,17 +1,49 @@
...
+          className="userid"
+          placeholder="User ID"
+          onChange={(e) =>
+            setLoginFormData({ ...loginFormData, email: e.target.value })
+          }
+        />
+        <input
+          type="password"
+          name="passwd"
+          id=""
```
Comment (sushil-mod) [inline src/pages/authpages/Login.js:38]: if you are not using id remove it

**Review by sushil-mod — APPROVED** [review]
Just a suggestion .Rest is good

**Pull request merged**